packages/ui/src/components/vuestic-components/va-button-dropdown/VaButtonDropdown.vue

```diff
@@ -5,8 +5,10 @@
       :disabled="disabled"
       :position="position"
       :offset="$props.offset"
-      @update:modelValue="toggleDropdown"
       :keep-anchor-width="keepAnchorWidth"
+      :close-on-click-inside="closeOnClickInside"
+      v-model="valueComputed"
```

Comment:
Doesn't make sense to use stateful mixin here I think.
If I understood correctly, we only need one state for "shown" dropdown. And that state is already present in dropdown.

So it's valid to pass `:stateful` down, but you can use `valueProxy` getter/setter instead of `valueComputed`.